resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -292,6 +292,14 @@ private[spark] object Config extends Logging {
       .stringConf
       .createOptional
 
+  val KUBERNETES_JOB_QUEUE = ConfigBuilder("spark.kubernetes.job.queue")
```

Comment:
As you know, if there is no other `job` config, this should be `jobQueue` instead of `job.queue`.

Reply:
I can get your concern in here, I think it's a good note. : )

The short plan is we need to also add `minCPU`/`priorityClassName` in here, and there are also some pontential job related introduce in future.